### PR TITLE
Smooth test run

### PR DESCRIPTION
**Pull request by utkarsharma2**

### CODE REVIEW

**Comment by dimberman** [comment]
Hi Utkarsh, can you give a bit more context on this PR in the description?

File: tests/operators/test_agnostic_aggregate_check.py [hunk 1]
```diff
@@ -147,16 +147,7 @@ def test_invalid_values(self):
...
-        aql.load_file(
-            path=str(self.cwd) + "/../data/homes_merge_1.csv",
-            output_table=Table(
-                "aggregate_check_test",
-                conn_id="snowflake_conn",
-                database="DWH_LEGACY",
-                schema="SANDBOX_AIRFLOW_TEST",
-            ),
-        ).operator.execute({"run_id": "foo"})
+    def test_postgres_exact_number_of_rows(self):
```
Comment (dimberman) [inline tests/operators/test_agnostic_aggregate_check.py:150]: Why did you change this from a snowflake test to a PG test?

File: tests/operators/test_snowflake_append.py [hunk 2]
```diff
@@ -103,30 +123,38 @@ def wait_for_task_finish(self, dr, task_id):
...
                 path=str(cwd) + "/../data/homes_main.csv",
-                output_table=Table(
-                    table_name="test_append_1",
-                    conn_id="snowflake_conn",
-                    schema="tmp_astro",
-                ),
+                file_conn_id="",
+                output_table=self.TABLE_1,
             )
             load_append = aql.load_file(
```
Comment (dimberman) [inline tests/operators/test_snowflake_append.py:148]: Can you do something similar hear to what you did in merge where you load a raw table into snowflake once and then just copy that table at test-time? I think that can save us some time on CI/CD as snowflake loads are comparatively slow.

File: tests/operators/test_snowflake_decorator.py [hunk 3]
```diff
@@ -73,15 +74,41 @@ class TestSnowflakeOperator(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.snowflake_table = test_utils.get_table_name(
+            "SNOWFLAKE_TRANSFORM_TEST_TABLE"
+        )
+        cls.snowflake_table_raw_sql = test_utils.get_table_name(
+            "SNOWFLAKE_TRANSFORM_RAW_SQL_TEST_TABLE"
+        )
```
Comment (dimberman) [inline tests/operators/test_snowflake_decorator.py:82]: I had no idea we could define variables at a class level like this. Great find!

File: tests/operators/test_agnostic_load_file.py [hunk 4]
```diff
@@ -30,6 +30,7 @@
 import unittest.mock
 
 import pandas as pd
+import utils as test_utils
```
Comment (dimberman) [inline tests/operators/test_agnostic_load_file.py:33]:
```suggestion
from ..operators import utils as test_utils
```

Making this change as IDEs can't recognize this import

File: tests/operators/test_dataframe.py [hunk 5]
```diff
@@ -20,6 +20,7 @@
 from unittest import mock
 
 import pandas
+import utils as test_utils
```
Comment (dimberman) [inline tests/operators/test_dataframe.py:23]:
```suggestion
from ..operators import utils as test_utils
```

File: tests/operators/test_snowflake_append.py [hunk 6]
```diff
@@ -29,6 +29,7 @@
 import pathlib
 import unittest.mock
 
+import utils as test_utils
```
Comment (dimberman) [inline tests/operators/test_snowflake_append.py:32]:
```suggestion
from ..operators import utils as test_utils
```

File: tests/operators/test_snowflake_decorator.py [hunk 7]
```diff
@@ -29,6 +29,7 @@
 import pathlib
 import unittest.mock
 
+import utils as test_utils
```
Comment (dimberman) [inline tests/operators/test_snowflake_decorator.py:32]:
```suggestion
from ..operators import utils as test_utils
```

File: tests/operators/test_snowflake_merge.py [hunk 8]
```diff
@@ -26,6 +26,7 @@
 import pathlib
 import unittest.mock
 
+import utils as test_utils
```
Comment (dimberman) [inline tests/operators/test_snowflake_merge.py:29]:
```suggestion
from ..operators import utils as test_utils
```

File: tests/operators/test_snowflake_merge_func.py [hunk 9]
```diff
@@ -26,6 +26,7 @@
 import pathlib
 import unittest.mock
 
+import utils as test_utils
```
Comment (dimberman) [inline tests/operators/test_snowflake_merge_func.py:29]:
```suggestion
from ..operators import utils as test_utils
```

**Review by dimberman — CHANGES_REQUESTED** [review]
Overall this looks fantastic, few minor nits but great job!

File: tests/operators/test_agnostic_aggregate_check.py (hunk 1 above)
Comment (utkarsharma2) [reply tests/operators/test_agnostic_aggregate_check.py:150]: I saw some test which was using the PG but named snowflake, I guess I changed the wrong one. I'll revert this.

Comment (utkarsharma2) [reply tests/operators/test_agnostic_aggregate_check.py:150]: @dimberman, Test is using Postgres table and there was a table loaded into snowflake which was never used in test. So I thought the name was misleading.

File: tests/operators/test_snowflake_append.py (hunk 2 above)
Comment (utkarsharma2) [reply tests/operators/test_snowflake_append.py]: @dimberman, I have made the required changes.

File: tests/operators/test_snowflake_decorator.py (hunk 3 above)
Comment (utkarsharma2) [reply tests/operators/test_snowflake_decorator.py:82]: Thanks 👍
